sys/include/net/nanocoap.h

```diff
@@ -18,7 +18,7 @@
  * provides high and low level interfaces to CoAP options, including Block.
  *
  * nanocoap includes the core structs to store message information. It also
- * also provides support for sending and receiving messages, such as
+ * also provides functions to support sending and receiving messages, such as
```

Comment:
```suggestion
 * also provides helpers for use before sending of after receiving messages, such as
```

Reply:
Added this suggestion to #12231 since it's an improvement to generic nanocoap.